code/modules/mob/mob_defines.dm

```diff
@@ -343,6 +343,7 @@
 	var/disconnect_time = null
 
 	var/atom/movable/screen/shadekin/shadekin_display = null
+	var/atom/movable/screen/crew_shadekin/crew_shadekin_display = null
```

Comment:
why cant you use the shadekin_display for crewkin? Answer in 5 sentences or less

Reply:
I can't. Removing it from mob_defines, human_hud/human, and changing references to it in shadekin_blackeyed back to the normal shadekin_display.